docs/blinkid-verify/OssDependencies.md

```
@@ -138,3 +138,41 @@ This report includes:
 | use-sync-external-store | 1.2.2 | MIT | N/A | [Link](https://github.com/facebook/react#readme) |
 | wasm-feature-detect | 1.8.0 | Apache-2.0 | Surma | [Link](https://github.com/GoogleChromeLabs/wasm-feature-detect#readme) |
 | zustand | 4.5.5 | MIT | Paul Henschel | [Link](https://github.com/pmndrs/zustand) |
+
+## C++ Dependencies
+
+| Package | License | URL | Description |
+|---------|---------|-----|-------------|
+| abseil/20250512.0.1@microblink/main#120e2c1b766e76087248ee491d67cb52 | Apache-2.0 | [Link](https://github.com/microblink/abseil-cpp) | Abseil Common Libraries (C++) from Google |
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)